src/guide/transitions.md

```diff
@@ -35,7 +35,10 @@ When an element with transition is inserted or removed, Vue will:
 A typical CSS transition looks like this:
 
 ``` html
-<div v-if="show" transition="expand">hello</div>
+<div id="app">
+    <div v-if="show" transition="expand">hello</div>
+    <button @click="toggle">botao</button>
```

Comment:
Use **English** language, @jhserodio ! I don't understand what does **botao** mean. - This is what Evan means.